polkadot-parachains/src/command.rs

```diff
@@ -188,8 +173,7 @@ impl SubstrateCli for RelayChainCli {
 macro_rules! construct_partials {
 	($config:expr, |$partials:ident| $code:expr) => {
 		if $config.chain_spec.is_shell() {
-			let $partials = crate::service_shell::new_partial(&$config)?;
-			$code
+			unimplemented!()
```

Comment:
Don't we want to remove that branch?

Reply:
the whole code should disappear, so I won't clean up. omni-node form now on